Plugins/Public/Laz/Main.cpp

```diff
@@ -0,0 +1,279 @@
+// Laz's FLHook Plugin - Template by Alley
+// Current Features:
```

Comment:
Not going down the "let's make one plugin hold every feature written and completely destroy the concept of the plugin system" route again